test_tortoise_interpreter.py

```diff
@@ -4,7 +4,12 @@
     main_tortoise,
     sanitize_program,
     navigate_command_functions,
-    pen_up, pen_colour,
+    pen_up,
+    pen_down,
+    pen_colour,
+    get_pen_state,
+    move_east,
+    move_north
```

Comment:
Why have we imported these functions? Their behavior could already be tested through `main_tortoise`.

Reply:
we need them for the test_navigate_command_functions test